### ISSUE TITLE
Coolwalk card layout possibility

### ISSUE THREAD

**Issue by plapic** [AndreyPavlenko/Fermata#272]
Not sure how this happened but its the perfect size for media card layout while using the larger card area for navigating. 
![screendump - 2023-07-07-07_02_05](https://github.com/AndreyPavlenko/Fermata/assets/45576029/257f2132-5cb0-4739-b42d-f477e422dfc6)

**Comment by HOSTED-POWER** [comment]
Yes, I wonder how it works? Atm Fermata is always "taking over" waze. It's not handled as a media player, but rather as full screen navigation app like waze. How can this behaviour be influenced? Once I launch Fermata I lose my waze screen ...

**Comment by plapic** [comment]
> Yes, I wonder how it works? Atm Fermata is always "taking over" waze. It's not handled as a media player, but rather as full screen navigation app like waze. How can this behaviour be influenced? Once I launch Fermata I lose my waze screen ...

Yes it would be great if Fermata could be treated as a media player and support small card coolwalk format. The small size as shown in the hiccup I had is still usable.  At least you still hear Fermata audio in the background when Waze is in the foreground.

**Comment by HOSTED-POWER** [comment]
Yes or if waze could be set as the "small" application, for me it would be fine too. Not sure how this all works underlying. But an option to present fermata as media player would be good. I'm just not sure if it can still play video's then, I don't want to miss seeing the video's. 

( So Ideally  I see waze + video from youtube at the same time)
